solutions/cses-1640.mdx

```diff
@@ -21,18 +21,18 @@ exist?
 
 <PySection>
 
-# Using a Dictionary
+## Using a Dictionary
```

Comment:
Maybe use three #'s (`###`) the dictionary/map headers because they are still part of the solution 1 section